### PR TITLE
Update UG, PPP, AboutUs

### PR DESCRIPTION
**Pull request by jingjie88**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -71,31 +71,31 @@ Shows a message explaning how to access the help page.
...
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 A person can have any number of tags (including 0)
 </div>
 
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01, j/software engineer`
+* `add n/Betsy Crowe e/betsycrowe@example.com a/Street street p/1234567 t/good interviewee j/data analyst`
```
Comment (Jonyxzx) [inline docs/UserGuide.md:86]: Your format is job before tag but the example is tag before job. Not sure it might mess with the parsing or not

**Review by Jonyxzx — APPROVED** [review]
Remember to use the automatic close in your PR description (e.g. close #XX) for automatic issue closer. Overall LGTM!

File: docs/images/jingjie88.png
Comment (kiwibang) [inline docs/images/jingjie88.png:1]: Handsome man. +1 awesome points!

File: docs/team/jingjie88.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
```
Comment (kiwibang) [inline docs/team/jingjie88.md:1]: Layout follows standard. Very nice!

File: docs/images/jingjie88.png
Comment (kiwibang) [inline docs/images/jingjie88.png:1]: Very handsome. +1 banan! 🍌 🦍

File: docs/UserGuide.md (hunk 1 above)
Comment (jingjie88) [reply docs/UserGuide.md:86]: Thanks for the comment! If I'm not wrong the original ab3 user guide also placed the tag at the front to show that the order doesn't matter. I've tested it and it seems like the order of all the fields can be switched.

File: docs/AboutUs.md [hunk 3]
```diff
@@ -18,15 +18,15 @@ We currently do not offer any means to contacting us, thank you for your underst
...
 
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](http://github.com/jingjie88)]
+[[portfolio](team/jingjie88.md)]
 
-* Role: Team Lead
-* Responsibilities: UI
+* Role: Developer
+* Responsibilities: Improve `add`, `edit`, `delete` features
```
Comment (kiwibang) [inline docs/AboutUs.md:29]: Perhaps can be more specific with the job roles? 🤔

**Review by kiwibang — COMMENTED** [review]
LGTM! Just a small nit to fix. 👍

**Pull request merged**